datadog_checks_base/datadog_checks/checks/openmetrics/mixins.py

```diff
@@ -496,12 +496,15 @@ def _submit_gauges_from_summary(self, metric_name, metric, scraper_config, hostn
                 self.log.debug("Metric value is not supported for metric {}".format(sample[self.SAMPLE_NAME]))
                 continue
             custom_hostname = self._get_hostname(hostname, sample, scraper_config)
-            tags = self._metric_tags(metric_name, val, sample, scraper_config, hostname=custom_hostname)
             if sample[self.SAMPLE_NAME].endswith("_sum"):
+                tags = self._metric_tags(metric_name, val, sample, scraper_config, hostname=custom_hostname)
```

Comment:
Let's move this outside of the `if` and then delete the other `tags` variables just to keep it clean.